src/kerberos/server/as_exchange.rs

```diff
@@ -71,7 +71,7 @@ pub async fn request_tgt(
         cname_type,
         snames: &[TGT_SERVICE_NAME, &realm],
         // 4 = size of u32
-        nonce: &OsRng.gen::<[u8; 4]>(),
+        nonce: &OsRng.try_next_u32()?.to_be_bytes(),
```

Comment:
note: It’s probably fine to switch to StdRng (ChaCha20) for most things: https://docs.rs/rand/latest/rand/rngs/struct.StdRng.html

Reply:
Agree. I didn't do it because it was not necessary. Do you want me to do it in this PR?

Reply:
Done